### PR TITLE
Azul Readme: notes about branch merging

### PR DESCRIPTION
**Pull request by amarjandu**
adds note about the branch merging section being out of date in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -597,6 +597,8 @@ deploy`) in the affected deployment.
 
 # 6. Branch flow & development process
 
+## Note: The section below regarding branch management is out-of-date.
```
Comment (hannes-ucsc) [inline README.md]: I wouldn't use a heading style for the note. IIRC, GH markdown does not support note boxes. If that's the case, I would use just bold.

```suggestion
## Note: This section should be considered a draft. It describes a future extension to the current branching flow.
```

Comment (amarjandu) [reply README.md]: went with bold && used suggested message change.

**Comment by hannes-ucsc** [comment]
@amarjandu For !fixup commits to work as intended, the string that follows "!fixup" needs to exactly match the title of a previous commit. Otherwise, git --auto-squash won't know what original commit to squash the fixup commit with. The easiest way to achieve that is via `git commit --fixup` or right clicking on a commit in the change log view in PyCharm.

**Comment by amarjandu** [comment]
> @amarjandu For !fixup commits to work as intended, the string that follows "!fixup" needs to exactly match the title of a previous commit. Otherwise, git --auto-squash won't know what original commit to squash the fixup commit with. The easiest way to achieve that is via `git commit --fixup` or right clicking on a commit in the change log view in PyCharm.

👍  thanks for letting me know i'll use `git commit --fixup` next time.

**Pull request merged**